Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
```

Comment:
With this typo fix, we are no longer bug-compatible with [upstream](https://github.com/coreos/coreos-vagrant/blob/master/Vagrantfile#L48). :smiling_imp: